app/components/Snaps/SnapUICard/SnapUICard.tsx

```diff
@@ -41,7 +41,7 @@ export const SnapUICard: FunctionComponent<SnapUICardProps> = ({
       alignItems={AlignItems.center}
     >
       {image && (
-        <SnapUIImage width={32} height={32} borderRadius={999} value={image} />
+        <SnapUIImage width={32} height={32} borderRadius={100} value={image} />
```

Comment:
```suggestion
        <SnapUIImage width={32} height={32} borderRadius={999} value={image} />
```
IMO just keep this as-is, 999 matches extension

Reply:
Sure, then we should update the extension to match this instead